sample-applications/blazing-coffee/BlazingCoffee/Client/Pages/Index.razor

```diff
@@ -1,67 +1,146 @@
+                    <Columns>
+                        <GridColumn Field="@nameof(Employee.Rating)" Width="110px" Title="@L["Employee_Grid_Title_Rating"]">
+                            <Template>
+                                <Rating CssClass="rating-readonly" Value="((Employee)context).Rating" />
+                            </Template>
+                        </GridColumn>
+                        <GridColumn Field="@nameof(Employee.Target)" Width="230px" Title="@L["Employee_Grid_Title_Engagement"]">
+                            <Template>
+                                @{
+                                    var employee = context as Employee;
+                                    if (employee != null)
+                                    {
+                                       <TelerikProgressBar Max="100" Value="employee.Target" />
+                                    }
+                                }
+                            </Template>
+                        </GridColumn>
+                        <GridColumn Field="@nameof(Employee.Budget)" DisplayFormat="{0:C}" Width="100px" Title="@L["Employee_Grid_Title_Budget"]"></GridColumn>
+                    </Columns>
+                </GridColumn>
+                <GridColumn Title="@L["Employee_Grid_Title_Contacts"]">
+                    <Columns>
+                        <GridColumn Field="@nameof(Employee.Phone)" Width="130px" Title="@L["Employee_Grid_Title_Phone"]"></GridColumn>
+                        <GridColumn Field="@nameof(Employee.Address)" DisplayFormat="{0:C}" Width="200px" Title="@L["Employee_Grid_Title_Address"]"></GridColumn>
+                    </Columns>
+                </GridColumn>
+            </GridColumns>
+        </TelerikGrid>
     }
+</CardContainer>
+
+    @code {
+    private Employee[] employees;
+    TelerikGrid<Employee> Grid { get; set; }
+    public bool SingleTeamSelected { get; set; }
+    public bool AllTeamsSelected { get; set; } = true;
 
     protected override void OnInitialized()
     {
-        Layout.DocsTitle = L["Home"];
+        Layout.DocsTitle = L["HumanCapital"];
 
         base.OnInitialized();
     }
 
-    protected override async Task OnAfterRenderAsync(bool firstRender)
+    protected override async Task OnInitializedAsync()
+    {
+        employees = await Http.GetFromJsonAsync<Employee[]>("api/employees");
+    }
+
+    private void OnCellRender(GridCellRenderEventArgs args)
+    {
+        args.Class = "center-cell";
+    }
+
+    public async Task OnButtonGroupClick()
     {
-        if (firstRender)
-        {
-            var state = await localStorage.GetItemAsync<TileLayoutState>("IndexLayout");
-            if (state != null)
-            {
-                tileLayout.SetState(state);
-            }
+        var state = Grid.GetState();
+
+        if(AllTeamsSelected){
+            state.FilterDescriptors.Add(SingleTeamDescriptor());
+        } else {
+            state = null;       
         }
+        
```

Comment:
```suggestion
        if (AllTeamsSelected)
        {
            state.FilterDescriptors.Add(SingleTeamDescriptor());
        } 
        else 
        {
            state = null;       
        }
        
```
A general .NET code formatting can be used instead of JavaScript-ish.